tests/foreman/cli/test_logging.py

```diff
@@ -0,0 +1,251 @@
+"""CLI tests for logging.
+
+:Requirement: Logging
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: logging
+
+:TestType: Functional
+
+:CaseImportance: Medium
+
+:Upstream: No
+"""
+import re
+from fauxfactory import gen_string
+from robottelo import ssh
+from robottelo.ssh import upload_file
+from robottelo import manifests
+from robottelo.test import CLITestCase
+from robottelo.cli.factory import (
+    make_org,
+    make_product_wait,
+    make_repository,
+)
+from robottelo.cli.subscription import Subscription
+
+
+class SimpleLoggingTestCase(CLITestCase):
+    """Test class for default logging to files."""
+
+    org = None
+    product = None
+
+    @classmethod
+    def setUpClass(cls):
+        """Tests for logging to files"""
+        super(SimpleLoggingTestCase, cls).setUpClass()
+
+        # need own org for the manifest refresh test
+
+        cls.org = make_org(cached=True)
+        cls.product = make_product_wait(
+            {u'organization-id': cls.org['id']},
+            )
+
+    def _make_repository(self, options=None):
+        """Makes a new repository and asserts its success"""
+        if options is None:
+            options = {}
+
+        if not options.get('product-id'):
+            options[u'product-id'] = self.product['id']
+
+        return make_repository(options)
+
+    def test_positive_logging_from_foreman_core(self):
+        """Check that GET command to Hosts API is logged and has request ID.
+
+        :id: 0785260d-cb81-4351-a7cb-d7841335e2de
+
+        :expectedresults: line of log with GET has request ID
+
+        :CaseImportance: Medium
+        """
+
+        GET_line_found = False
+        source_log = '/var/log/foreman/production.log'
```

Comment:
source_log is used multiple times, you can either have in a class or module level or keep in constant.py.

Reply:
But in the test for candlepin its not the same. Also, future tests will have different source.